price-server/src/fee_calculator/config.rs

```diff
@@ -36,21 +37,11 @@ impl Default for FeeCalculatorConfig {
 #[cfg(test)]
 mod tests {
     use super::*;
-    use rust_decimal::prelude::*;
 
     #[test]
     fn defaults() {
-        assert_eq!(
-            default_base_fee_rate(),
-            Decimal::from_str("0.0005").unwrap()
-        );
-        assert_eq!(
-            default_immediate_fee_rate(),
-            Decimal::from_str("0.0005").unwrap()
-        );
-        assert_eq!(
-            default_delayed_fee_rate(),
-            Decimal::from_str("0.0007").unwrap()
-        );
+        assert_eq!(default_base_fee_rate(), dec!(0.0005));
```

Comment:
This test is no longer any good since it is just restating the implementation. Perhaps use `from_str` here on the right side.